test/simulator/timestamp/src/session_simulator.cpp

```diff
@@ -27,11 +27,13 @@
  */
 
 #include "session_simulator.h"
-#include "error_simulator.h"
 
 #include <cassert>
 #include <iostream>
 
+#include "error_simulator.h"
+#include "timestamp_manager.h"
+
 session_simulator::session_simulator() : _txn_running(false) {}
```

Comment:
We should init `_has_commit_ts` too. Make sure the init order is the same as the declaration order.
```suggestion
session_simulator::session_simulator() : _has_commit_ts(false), _txn_running(false) {}
```

Reply:
Fixed.